tests/TestCase/Model/Table/QueuedTasksTableTest.php

```diff
@@ -458,20 +464,20 @@ public function testRequestGroup() {
 		$this->assertEquals(6, unserialize($tmp['data']));
 
 		// use FindProgress on the testgroup:
-		$progress = $this->QueuedTasks->find('progress', [
+		$progress = $this->QueuedTasks->find('all', [
 			'conditions' => [
-				'group' => 'testgroup'
+				'task_group' => 'testgroup'
 			]
-		]);
+		])->toArray();
 
 		$this->assertEquals(3, count($progress));
 
 		$this->assertNull($progress[0]['reference']);
-		$this->assertEquals($progress[0]['status'], 'IN_PROGRESS');
+#		$this->assertEquals($progress[0]['status'], 'IN_PROGRESS');
 		$this->assertEquals($progress[1]['reference'], 'Job number 4');
-		$this->assertEquals($progress[1]['status'], 'IN_PROGRESS');
+#		$this->assertEquals($progress[1]['status'], 'IN_PROGRESS');
 		$this->assertEquals($progress[2]['reference'], 'Job number 6');
-		$this->assertEquals($progress[2]['status'], 'IN_PROGRESS');
+#		$this->assertEquals($progress[2]['status'], 'IN_PROGRESS');
```

Comment:
nop, the field status will never be filled while executing, I've added the status field with my additional migration that it exists in the database